packages/backend/src/controllers/tasks/dto/create-task.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { TaskModel } from '../../../db/task/task.schema';
+
+export class CreateTaskDto extends TaskModel {
+  @ApiProperty()
+  readonly name: string;
+
+  @ApiProperty()
+  readonly description: string;
+
+  @ApiProperty()
+  readonly due_date: Date;
+
+  @ApiPropertyOptional()
+  readonly interval: number;
+
+  @ApiProperty()
+  readonly pool: string[];
```

Comment:
at a glance i don't really understand this property. maybe a documentation comment on either this property (or `createTaskDto`) should be created?